development/imagebump-detector/pkg/common.go

```diff
@@ -0,0 +1,49 @@
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+
+	"gopkg.in/yaml.v3"
+)
+
+func ParseNotationFile(filePath string) (string, string, error) {
+	f, err := os.Open(filePath)
+	defer f.Close()
+	if err != nil {
+		panic(err)
```

Comment:
Function should not `panic()`. You should rather return an error with proper error message:

```suggestion
		return nil, nil, err
```